src/glrm.jl

```diff
@@ -268,7 +274,7 @@ function fit!(glrm::GLRM; params::Params=Params(), ch::ConvergenceHistory=Conver
     t = time() - t
     update!(ch, t, ch.objective[end])
 
-    return glrm.X, glrm.Y, ch
+    return glrm.X, glrm.Y, ch, alpha[1]
```

Comment:
why return the step size? this breaks any code that currently calls `X, Y, ch = fit!(...)`. it would be better to add `alpha` as a new field in the `ConvergenceHistory` object, if you think it should be returned.